src/index.html

```diff
@@ -14,7 +14,7 @@ <h1>Leaderboard</h1>
     <h4>Recent scores</h4>
   <button class="refresh" type="reset" id="refresh-button">Refresh</button></div>
   <div id="list-form-container">
-    <div id="list"></div>
+    <div id="scores-list"></div>
```

Comment:
- [ ] I added my score to the app but I cannot see the score unless I go to the browser console and inspect it (as you can see in the image). Users who use the app should be able to see their scores. Kindly redesign your UI and ensure users can see their scores.

  ![image](https://user-images.githubusercontent.com/81468675/217328582-e4a99dd5-b376-4987-8e9b-df7f6432bf63.png)

Reply:
Please try to enter like 5 scores and will you see some of them. I belive it is a problem with style

Reply:
I can see your score. Julie: 23

Reply:
Hi @djo1975, it would be nice to redesign your UI so that all scores can be seen. Even if you are not in the styling milestone, having a good user interface at wireframe level is great for design 👍

Reply:
Hi @Gambit142. You can see all scores now.

Reply:
> Hi @Gambit142. You can see all scores now.

Hi @djo1975, yea, I can see all scores. Great work and keep rocking 🚀. Happy coding 💻